bessctl/test_sugar.py

```diff
@@ -0,0 +1,100 @@
+import fnmatch
+import os
+import sugar
+import unittest
+
+
+class TestSugar(unittest.TestCase):
+    """
+    All scripts in conf/ will be dynamically added here as individual
```

Comment:
Adding Python coverage to codecov will be done in a separate PR (still figuring out)